src/core.js

```diff
@@ -171,11 +176,12 @@ export function begin() {
 		runLoggingCallbacks( "begin", {
 			totalTests: Test.count,
 			modules: modulesLog
+		} ).then( function() {
+			unblockAndAdvanceQueue();
```

Comment:
You don't need the outer wrapping function. You can just inline this.

```
.then( unblockAndAdvanceQueue );
```

Reply:
Goodcall, updated.